acvp_transport.c

```diff
@@ -54,12 +54,14 @@ static long acvp_curl_http_get (ACVP_CTX *ctx, char *url, void *writefunc)
     long http_code = 0;
     CURL *hnd;
     struct curl_slist *slist;
-    char bearer[1048]; //TODO: remove this from the stack
 
     slist = NULL;
     if (ctx->jwt_token) {
-        snprintf(bearer, 1048, "Authorization: Bearer %s", ctx->jwt_token);
+    	int buf_size = strlen(ctx->jwt_token) + 22;
```

Comment:
Let's just set it to 512 bytes for now.  The ACVP spec is no where near complete.  The content of the JWT may still be changing.